src/markdown-pages/nr1-cli/nr1-general.mdx

```diff
@@ -0,0 +1,166 @@
+---
+path: '/explore-docs/nr1-general'
+title: 'New Relic One CLI general commands'
+template: 'GuideTemplate'
+description: 'An overview of some general commands you can use with the New Relic One CLI.'
+tileShorthand:
+  title: 'New Relic One general commands'
```

Comment:
I was curious why we have a guide that is just a subset of commands for nr1 CLI, when we've documented the complete set of commands?  Is there some use case this is addressing?

Reply:
If this solves a user problem then maybe calling this Most common commands, instead of general.

Reply:
Good call -- I like this idea. I wasn't quite sure of the title either.